content_sync/pipelines/definitions/concourse/common/steps.py

```diff
@@ -326,3 +328,17 @@ def __init__(self, branch: str, short_id: str, **kwargs):
             ),
             **kwargs,
         )
+
+
+models = [
+    ClearCdnCacheStep,
+    ErrorHandlingStep,
+    OcwStudioWebhookStep,
+    OcwStudioWebhookCurlStep,
+    OpenCatalogWebhookStep,
+    SiteContentGitTaskStep,
+    SlackAlertStep,
+]
+
+for model in models:
+    model.model_rebuild()
```

Comment:
Similarly here, rather than calling this outside of the context of the class, we should be calling `self.model_rebuild()` within the `__init__` of the class itself. Calling it here, the uninstantiated class will have this called on it whenever anything from the file is imported.